src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ManageContent/ManageContentPageServiceTests.cs

```diff
@@ -367,6 +355,146 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
             MockUtils.VerifyAllMocks(dataBlockService, methodologyVersionRepository, releaseFileService);
         }
 
+        [Fact]
+        public async Task GetManageContentPageViewModel_IsPrerelease()
+        {
+            var publication = new Publication
+            {
+                Contact = new Contact(),
+                LegacyReleases = new List<LegacyRelease>(),
+                Slug = "test-publication",
+                Title = "Publication",
+                Topic = new Topic
+                {
+                    Theme = new Theme(),
+                }
+            };
+
+            var release = new Release
+            {
+                Id = Guid.NewGuid(),
+                NextReleaseDate = new PartialDate {Day = "9", Month = "9", Year = "2040"},
+                PreReleaseAccessList = "Test access list",
+                Publication = publication,
+                PublishScheduled = DateTime.Parse("2020-09-08T23:00:00.00Z", styles: DateTimeStyles.AdjustToUniversal),
+                Published = null,
+                ReleaseName = "2020",
+                RelatedInformation = new List<Link>(),
+                Slug = "2020-21",
+                TimePeriodCoverage = AcademicYear,
+                Type = ReleaseType.OfficialStatistics,
+                Updates = new List<Update>(),
+                KeyStatistics = new List<KeyStatistic>(),
```

Comment:
Some of the fields setup on this test can probably be slimmed down if they aren't relevant to the test.

Here's an example of two which are already initialised by default in `Release`.